domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
 smart-co.co.uk
 smartedgesolutions.co.uk
 smartsourcing.plc.uk
 snap.hr
+socode.co.uk
 softwarepersonnel.co.uk
 solagroup.com
 soprasteria.com
+soultek.co.uk
 sourced.ai
 sourced.tech
 southern-lights.co.uk
 spectrumit.co.uk
 speerhead.com
 spencer-rose.com
 spgsearch.com
+spherelondon.co.uk
```

Comment:
Welcome to Sphere Digital Recruitment.

We fill jobs across the entire digital Sphere.